.github/workflows/Cancel.yml

```diff
@@ -0,0 +1,17 @@
+name: Cancel
+ on:
+   workflow_run:
+     workflows:
+       - "CI"
+       - "IntegrationTest"
+       - "JuliaNightly"
+     types:
+       - requested
+ jobs:
+   cancel:
+     runs-on: ubuntu-latest
+     steps:
+     - uses: styfle/cancel-workflow-action@0.9.0
+       with:
+         all_but_latest: true
```

Comment:
By default, only the workflows started before this action are cancelled. This flag cancels all but the latest workflow, which is helpful if the pipeline is saturated